src/steps/check-pardot-tracking.ts

```diff
@@ -65,17 +66,22 @@ export class CheckPardotTrackingStep extends BaseStep implements StepInterface {
           matchingRequests.map(request => decodeURIComponent(request.url)).join('\n\n'),
         ]);
       }
-      return this.pass('Successfully detected Pardot Tracking request for account id %d, and campaign id %d.', [
-        aid,
-        cid,
-      ]);
+      const record = this.createRecord(evaluatedRequests[0].rawRequest._url);
+      return this.pass(
+        'Successfully detected Pardot Tracking request for account id %d, and campaign id %d.', [aid, cid], [record]);
     } catch (e) {
       return this.error('There was a problem checking Pardot Tracking request: %s', [
         e.toString(),
       ]);
     }
   }
 
+  private createRecord(url) {
+    let params = {};
```

Comment:
- [ ] Still seeing issues with the parameters having strange names.  Would probably be resolved by using those node built-ins rather than string munging:

![Screen Shot 2020-02-28 at 11 55 46 AM](https://user-images.githubusercontent.com/3496491/75583089-cee8b800-5a21-11ea-9918-9e57c167469b.png)